life_expectancy/loading_saving.py

```diff
@@ -1,29 +1,51 @@
+"""
+Module to load and save data
+"""
+
 from pathlib import Path
 
 import pandas as pd
 
+PARENT_PATH = Path(__file__).parent
+FILE_PATH = PARENT_PATH / 'data'
+TSV_FILE_NAME = 'eu_life_expectancy_raw.tsv'
+JSON_FILE_NAME = 'eurostat_life_expect.json'
+TSV_FILE_PATH = FILE_PATH / TSV_FILE_NAME
+JSON_FILE_PATH = FILE_PATH / JSON_FILE_NAME
+
 
-def load_data(file_path: str) -> pd.DataFrame:
+class DataLoader:
+    """
+    This class is responsible for verifying the format
+    and assign the right load method
     """
-    Method used to load data.
+    def __init__(self, data_format):
+        self.data_format = data_format
 
-    Args:
-        file_path [str]: file path to load data.
+    def load_csv(self) -> pd.DataFrame:
+        """Method to load csv file"""
+        return pd.read_csv(TSV_FILE_PATH, sep='\t')
 
-    Returns:
-        [pd.DataFrame]: Dataframe to be cleaned.
-    """
-    data_df = pd.read_csv(file_path, sep='\t')
+    def load_json(self) -> pd.DataFrame:
+        """Method to load json file"""
+        return pd.read_json(JSON_FILE_PATH)
+
+    def load_data(self) -> None:
```

Comment:
In my opinion the code would be cleaner if you removed the methods `load_csv` and `load_json`:
```python
def load_data(self) -> None:
        """Method to verify data format"""
        if self.data_format == 'csv':
            return pd.read_csv(TSV_FILE_PATH, sep='\t')
        if self.data_format == 'json':
            return pd.read_json(JSON_FILE_PATH)
        raise ValueError("Unsupported data format.")
```

Reply:
Resolved